### PR TITLE
Support to embedding Etherpad into tasks and projects - Fixes #727

### LINKED ISSUE #727: Request: etherpad-lite integration
Integration Etherpad-lite ( https://github.com/ether/etherpad-lite ) its possible with API ( https://github.com/ether/etherpad-lite/wiki/HTTP-API );

The proposal:
- Set configuration URL and token API of etherpad-lite in kanboard;
- All description or text-area using alternative method with etherpad-lite.

Example:  http://www.odoohelp.com/odoo-etherpad-integration-formerly-openerp/

### PR DESCRIPTION
**Pull request by SamuelMoraesF**
* Adding `Etherpad` field into `Projects > Edit`, `Tasks > Create`, `Tasks > Edit`
* Adding iframe to the Etherpad page(AKA pad) into `Project > Summary` and `Task > Summary`

**Features:**
* Etherpad Username as the same of the Kanboard user(and can be modified)

firefox_screenshot_2015-04-04t16-04-57 666z

### CODE REVIEW

File: app/Controller/Base.php [hunk 1]
```diff
@@ -146,13 +146,20 @@ public function __get($name)
      *
      * @access private
      */
-    private function sendHeaders($action)
+    private function sendHeaders($controller, $action)
     {
+        // The page have an IFRAME
```
Comment (SamuelMoraesF) [inline app/Controller/Base.php]: TBD, we need an best way to do it.

**Comment by valessiobrito** [comment]
Thanks a lot @SamuelMoraesF :+1:

**Comment by fguillot** [comment]
What is the use case for that?

**Comment by SamuelMoraesF** [comment]
Some tasks/projects need a draft/collaborative document, and Etherpad is an awesome tool to do it!

Also, Etherpad have a lot of plugins too and can be used to collaborative draw, files management, video/audio chat, slideshows, spreadsheets... an official list are disponible [here](http://static.etherpad.org/plugins.html).

Some examples that can be used with Etherpad support:

**Task:** ``Write post to announce the event on our blog`` (pad with the post content)
**Project:** Pad to organize meeting resume

**Comment by froschdesign** [comment]
:-1: 

This is a special use case and not a part for a kanban board.
It also implements external services and this is a no-go!

So, don't add this to the core.

**Comment by BlueTeck** [comment]
I agree :-1:

**Pull request closed without merging**